src/lib.rs

```diff
@@ -72,11 +72,11 @@ pub async fn run(args: Cli) -> Result<()> {
     let _monitor = Interrupt::run_ctrl_c_monitor();
     use Commands::{Build, EndToEnd, New, Serve, Test, Watch};
     match args.command {
-        New(_) => panic!(),
         Build(_) => command::build_all(&config).await,
         Serve(_) => command::serve(&config.current_project()?).await,
         Test(_) => command::test_all(&config).await,
         EndToEnd(_) => command::end2end_all(&config).await,
         Watch(_) => command::watch(&config.current_project()?).await,
+        New(_) => unreachable!(r#""new" command should have already been run"#),
```

Comment:
I figure `unreachable!` is closer to what is meant semantically here. The docs for that macro say that it's useful whenver the compiler can't determine if some code is unreachable; I'm unsure if that means it will aid the compiler in optimization, but it seems more appropriate regardless.